### PR TITLE
OSDOCS-9826: adds Y+2 update paths MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.16+

Issue:
[OSDOCS-9826](https://issues.redhat.com//browse/OSDOCS-9826)

Link to docs preview:
[Updates - all](https://73988--ocpdocs-pr.netlify.app/microshift/latest/microshift_updating/microshift-about-updates)

QE review:
- [x] QE has approved this change.

Additional info:
Release note: https://github.com/openshift/openshift-docs/pull/73997

### CODE REVIEW

File: snippets/microshift-update-paths-snip.adoc [hunk 1]
```diff
@@ -8,12 +8,18 @@
...
-* Generally Available Version 4.14 to 4.14.z or 4.15 on {op-system-ostree} 9.2 or 9.3
-* Generally Available Version 4.14 to 4.14.z or 4.15 on {op-system} 9.2 or 9.3
+{microshift-short} version 4.16::
+* Version 4.16 to 4.16.z on {op-system} or {op-system-ostree} 9.4
 
-* Generally Available Version 4.15 to 4.15.z on {op-system-ostree} 9.2 or 9.3
-* Generally Available Version 4.15 to 4.15.z on {op-system} 9.2 or 9.3
\ No newline at end of file
+{microshift-short} version 4.15::
+* Version 4.15 to 4.16 on {op-system} or {op-system-ostree} 9.4
```
Comment (dhellmann) [inline snippets/microshift-update-paths-snip.adoc:19]: We don't support 4.15 on 9.4, but this reads as though we might.

```suggestion
* Version 4.15 on {op-system} or {op-system-ostree} 9.2 or 9.3 to 4.16 on {op-system} or {op-system-ostree} 9.4
```

File: snippets/microshift-update-paths-snip.adoc [hunk 2]
```diff
@@ -8,12 +8,18 @@
...
+* Version 4.15 to 4.15.z on {op-system} or {op-system-ostree} 9.2 or 9.3
+
+{microshift-short} version 4.14::
+* Version 4.14 to 4.16 on {op-system} or {op-system-ostree} 9.4
```
Comment (dhellmann) [inline snippets/microshift-update-paths-snip.adoc:23]: Similar here

```suggestion
* Version 4.14 on {op-system} or {op-system-ostree} 9.2 or 9.3 to 4.16 on {op-system} or {op-system-ostree} 9.4
```

**Review by dhellmann — COMMENTED** [review]
This all looks good. I have a couple of suggestions inline for clarity, but feel free to ignore those.

/lgtm

File: microshift_updating/microshift-about-updates.adoc [hunk 3]
```diff
@@ -32,4 +36,9 @@ include::snippets/microshift-rhde-compatibility-table-snip.adoc[leveloffset=+1]
 === Manual RPM updates
 You can use the manual RPM update path to replace your existing version of {microshift-short}. The versions of {op-system} and {microshift-short} must be compatible. Ensuring system health and completing additional system backups are manual processes.
 
+[IMPORTANT]
+====
+Ensure that your version of {op-system-base} is updated as required when updating {microshift-short}, especially across two minor versions of {microshift-short}.
```
Comment (mburke5678) [inline microshift_updating/microshift-about-updates.adoc:41]:
```suggestion
Ensure that your version of {op-system-base} is updated as required when updating {microshift-short}, especially if you are updating across two minor versions of {microshift-short}.
```

Comment (mburke5678) [reply microshift_updating/microshift-about-updates.adoc:41]: The _updated when updating_ sounds a bit weird.

```suggestion
Ensure that your version of {op-system-base} is up-to-date as required when updating {microshift-short}, especially across two minor versions of {microshift-short}.
```

File: microshift_updating/microshift-update-options.adoc [hunk 4]
```diff
@@ -14,13 +14,13 @@ include::snippets/microshift-update-paths-snip.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
-Updates of {microshift-short} from one minor version to the next must be in sequence. For example, you cannot update from 4.14 to 4.16. You must update 4.14 to 4.15 and so on.
+Updates of {microshift-short} are supported with a maximum of two minor versions. For example, you can update from 4.14 to 4.16 in a single step. Updating 4.14 to 4.15 first is not required.
```
Comment (mburke5678) [inline microshift_updating/microshift-update-options.adoc:17]:
```suggestion
Updates of {microshift-short} are supported with a maximum of two minor versions. For example, you can update from 4.14 to 4.16 in a single step without updating to 4.15 first .
```

File: microshift_updating/microshift-update-options.adoc [hunk 5]
```diff
@@ -14,13 +14,13 @@ include::snippets/microshift-update-paths-snip.adoc[leveloffset=+1]
...
 ====
 
 [id="microshift-update-options-standalone-updates_{context}"]
 == Standalone {microshift-short} updates
 
-You can update {microshift-short} without reinstalling your applications. {op-system} or {op-system-ostree} updates are also not required to update {microshift-short}, as long as the existing operating system is compatible with the new version of {microshift-short} that you want to use.
+You can potentially update {microshift-short} without reinstalling your applications and Operators. {op-system} or {op-system-ostree} updates are only required to update {microshift-short} if the existing operating system is not compatible with the new version of {microshift-short} that you want to use.
```
Comment (mburke5678) [inline microshift_updating/microshift-update-options.adoc:23]:
```suggestion
You can potentially update {microshift-short} without reinstalling your applications and Operators. {op-system} or {op-system-ostree} updates are only required before updating {microshift-short} if the existing operating system is not compatible with the new version of {microshift-short} that you want to use.
```

Comment (mburke5678) [reply microshift_updating/microshift-update-options.adoc:23]: Under what conditions can update without reinstalling apps or operators. The second sentence doesn't seem to answer this question. (Or, maybe it does and I am ignorant here.).

File: microshift_updating/microshift-update-rpms-manually.adoc [hunk 6]
```diff
@@ -10,10 +10,13 @@ Updating {product-title} for non-OSTree systems such as {op-system-base-full} re
 
 [IMPORTANT]
 ====
-{microshift-short} updates are supported from one minor version to the next in sequence. For example, you must update 4.14 to 4.15.
+{microshift-short} updates are supported from a maximum of two minor versions. For example, you can update 4.14 to 4.16 in a single step. Updating 4.14 to 4.15 first is not required.
```
Comment (mburke5678) [inline microshift_updating/microshift-update-rpms-manually.adoc:13]: I think the earlier note reads better. 
```suggestion
Updates of {microshift-short} are supported with a maximum of two minor versions. For example, you can update from 4.14 to 4.16 in a single step without updating to 4.15 first .
```

**Comment by mburke5678** [comment]
@ShaunaDiaz A few comments. Otherwise LGTM

File: microshift_updating/microshift-update-options.adoc (hunk 5 above)
Comment (ShaunaDiaz) [reply microshift_updating/microshift-update-options.adoc:23]: Separate thoughts, changed the structure a bit. In reality, we can'r answer that question here, customer has to figure that out based on custom apps and whatnot.

**Pull request merged**